README.md

```diff
@@ -60,36 +45,30 @@ A build toolchain for Snyk Docker images.
 | snyk/snyk:maven              | maven                                   |
 | snyk/snyk:maven-3-jdk-11     | maven:3-jdk-11                          |
 | snyk/snyk:maven-3-jdk-17     | maven:3-eclipse-temurin-17              |
-| snyk/snyk:maven-3-jdk-19     | maven:3-eclipse-temurin-19              |
 | snyk/snyk:maven-3-jdk-20     | maven:3-eclipse-temurin-20              |
 | snyk/snyk:maven-3-jdk-21     | maven:3-eclipse-temurin-21              |
 | snyk/snyk:maven-3-jdk-8      | maven:3-jdk-8                           |
 | snyk/snyk:dotnet             | mcr.microsoft.com/dotnet/core/sdk       |
-| snyk/snyk:dotnet-3.0         | mcr.microsoft.com/dotnet/core/sdk:3.0   |
-| snyk/snyk:dotnet-3.1         | mcr.microsoft.com/dotnet/core/sdk:3.1   |
 | snyk/snyk:node               | node                                    |
-| snyk/snyk:node-14            | node:14                                 |
-| snyk/snyk:node-16            | node:16                                 |
 | snyk/snyk:node-18            | node:18                                 |
 | snyk/snyk:node-20            | node:20                                 |
 | snyk/snyk:python             | python                                  |
-| snyk/snyk:python-2.7         | python:2.7                              |
-| snyk/snyk:python-3.6         | python:3.6                              |
-| snyk/snyk:python-3.7         | python:3.7                              |
 | snyk/snyk:python-3.8         | python:3.8                              |
 | snyk/snyk:python-3.9         | python:3.9                              |
 | snyk/snyk:python-3.10        | python:3.10                             |
 | snyk/snyk:python-3.11        | python:3.11                             |
 | snyk/snyk:python-alpine      | python:alpine                           |
 | snyk/snyk:ruby               | ruby                                    |
-| snyk/snyk:ruby-2.4           | ruby:2.4                                |
-| snyk/snyk:ruby-2.5           | ruby:2.5                                |
-| snyk/snyk:ruby-2.6           | ruby:2.6                                |
-| snyk/snyk:ruby-2.7           | ruby:2.7                                |
 | snyk/snyk:ruby-alpine        | ruby:alpine                             |
 | snyk/snyk:linux              | ubuntu                                  |
 
-### Deprecated images (not built and updated anymore)
+### Images not recommended to use
+We do not recommend using the following images as they are not supported by their vendors anymore, and we will deprecate them in the future.
+we will share a deprecation schedule in due time. 
+
+#### Following images are neither built nor maintained anymore by us
+The following images are due to be deprecated in the future. 
+
```

Comment:
Can we combine this and the list below. There are:
- Images Snyk builds, because the upstream image base is supported by their vendor
- Images Snyk no longer builds, because they are not supported by their vendor

I don't see the point of an in-between here.

Reply:
Agreed. It's two lists, one you should use, and one you shouldn't.

Reply:
@cmars no, we cannot. Let's disagree, agreeably here, and complete customer comms first. Thank you.